src/RwaConduit.sol

```diff
@@ -0,0 +1,100 @@
+pragma solidity 0.5.12;
+
+import "lib/dss-interfaces/src/dapp/DSTokenAbstract.sol";
+
+contract RwaConduit {
+    DSTokenAbstract public gov;
+    DSTokenAbstract public dai;
+    address public to;
+
+    event Push(address to, uint256 wad);
+
+    constructor(address _gov, address _dai, address _to) public {
+        gov = DSTokenAbstract(_gov);
+        dai = DSTokenAbstract(_dai);
+        to = _to;
+    }
+
+    function push() external {
+        require(gov.balanceOf(msg.sender) > 0);
```

Comment:
If the goal is to determine caller is a MKR governor, you may also want to check for the presence of `iou` tokens.

Reply:
good point, though note that the check is mostly symbolic (due to the 1 wei threshold), so unclear if it's worth adding another call as well as a dependency for tracking `iou` address (which can change with chief updates etc.)